app/code/Magento/MediaGalleryUi/view/adminhtml/web/js/image/image-edit.js

```diff
@@ -54,9 +55,33 @@ define([
         initialize: function () {
             this._super().initView();
 
+            this.removeMouseOverEvent();
+
             return this;
         },
 
+        /**
+         * Remove mousemove event from ui-select as it deprecated
```

Comment:
I think we should remove the mousemove event from `ui-select` component. It's deprecated since 2016

Reply:
ok